autorandr.py

```diff
@@ -715,6 +716,20 @@ def generate_virtual_profile(configuration, modes, profile_name):
                 shift += int(mode[shift_index])
             else:
                 configuration[output].options["off"] = None
+    elif profile_name == "clone":
+        biggest_width_resolution = sorted([output_modes[0] for output, output_modes in modes.items()], key=lambda x: x["width"], reverse=True)[0]
+        for output in configuration:
+            configuration[output].options = {}
+            if output in modes and configuration[output].edid:
+                mode = sorted(modes[output], key=lambda a: int(a["width"])*int(a["height"]) + (10**6 if a["preferred"] else 0))[-1]
```

Comment:
Would it be a good idea to prefer modes that have the same aspect ratio as `biggest_width_resolution`? Or: Would it be possible to maintain the aspect ratio when scaling below, adding black borders around the view? (idk myself, but making the screen a little larger than required and panning everything to the center might work?!)